pkg/aggregator/aggregator.go

```diff
@@ -257,3 +267,71 @@ func mergeSpecs(dest, source *spec.Swagger, renameModelConflicts, ignorePathConf
 	}
 	return nil
 }
+
+// deepEqualDefinitionsModuloGVKs compares s1 and s2, but ignores the x-kubernetes-group-version-kind extension.
+func deepEqualDefinitionsModuloGVKs(s1, s2 *spec.Schema) bool {
+	if !reflect.DeepEqual(s1.Extensions, s2.Extensions) {
+		if s1.Extensions != nil {
+			shallowCopy := *s1
+			s1 = &shallowCopy
+			s1.Extensions = nil
```

Comment:
it's okay to do so now, but we need to keep in mind that this function is actually `Modulo"Extensions"`. New extension key could be dropped silently during merging if we add any in future

Reply:
fixed. And added tests.